factorio-docker-watchdog.go

```diff
@@ -136,24 +164,19 @@ func updateVersion(version semver.Version) {
 	//}
 	//logrus.Info("Checkout branch ", version)
 
-	checksum, err := factorioGetChecksum(fmt.Sprintf("https://www.factorio.com/get-download/%s/headless/linux64", version))
-	if err != nil {
-		logrus.Panic(err)
-	}
-	logrus.Info("Got checksum ", checksum)
+	//err = editReadme(pathRepo, buildinfo)
```

Comment:
This would lead to the README getting outdated.

We should therefore either leave out specific information out of the README update the README accordingly.

Reply:
The question here would be, would it make sense to build the README before publishing it on docker hub?
Or should i edit it in the repo directly (like it did before)?

Reply:
Current behavior is to update the README before pushing to Dockerhub (see https://github.com/factoriotools/factorio-docker/commit/3f3fa0a00898d5bf038985a34a4ec9afbeda347e) . I think this is fine as we have an integration with GitHub Actions and the commit would _trigger_ the update of the tags on the Dockerhub, such that the information should only be outdated for a few minutes.

Reply:
> Or should i edit it in the repo directly (like it did before)?

Why should we do such a simple task manually?

Reply:
Not manually, i was thinking about leaving some kind of template syntax in the readme and on docker description update it will generate a full readable readme out of the buildinfo.json

Reply:
https://github.com/factoriotools/factorio-docker/pull/382/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3

Take a look here, I already added that template syntax

Reply:
Yea i saw that, was thinking about using those begin and end placeholders too

Reply:
The tags are now build by buildinfo.json
https://github.com/Fank/factorio-docker/commit/ffff85f59fc43b1bfa33a65b40c385895a305062